stories/core_components/GeoPushpinChart.tsx

```diff
@@ -196,7 +200,23 @@ storiesOf("Core components/GeoPushpinChart", module)
                 config,
             }),
         );
-    });
+    })
+    .add("with drillable items", () =>
+        screenshotWrap(
+            renderGeoPushpinChart({
+                projectId: "storybook",
+                config: DEFAULT_CONFIG,
+                drillableItems: [HeaderPredicateFactory.uriMatch("/gdc/md/storybook/obj/23/elements?id=4")],
+                location: ATTRIBUTE_LOCATION_GEOCHART,
+                size: MEASURE_SIZE_GEOCHART,
+                color: MEASURE_COLOR_GEOCHART,
+                segmentBy: ATTRIBUTE_SEGMENT_GEOCHART,
+                onDrill: (drillData: IDrillEventExtended) => {
+                    action("drill")(drillData);
+                },
```

Comment:
It could be just `onDrill: action("drill"),`

Reply:
updated